aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -142,4 +142,45 @@ Tensor flipud(const Tensor& self) {
   return self.flip({0});
 }
 
+Tensor moveaxis(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
```

Comment:
I feel like this should be in TensorShape.cpp. I skimmed through this file and it appears to be for operators that create a new tensor; most operators that create views are in TensorShape.cpp

Reply:
Have moved it to TensorShape.